packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig

```diff
@@ -29,11 +46,9 @@
         <span>{{suffix}}</span>
       {% endif %}
     </h{{ headingLevel }}>
-  {% if accordion %}
-    </button>
-  {% else %}
     </header>
   {% endif %}
+
   {% if actionStep.richText or actionStep.decorativeLink or actionStep.downloadLinks or actionStep.content %}
     <div class="ma__action-step__content {{ accordion ? 'js-accordion-content' : '' }}">

```

Comment:
expecting the accordion id, to go with `aria-controls`

Reply:
Fixed.